pkg/option/config.go

```diff
@@ -151,6 +151,10 @@ const (
 	// K8sRequireIPv6PodCIDRName is the name of the K8sRequireIPv6PodCIDR option
 	K8sRequireIPv6PodCIDRName = "k8s-require-ipv6-pod-cidr"
 
+	// K8sUseJSONPatch when set, uses JSON Patch to update CNP and CEP
+	// status in kube-apiserver.
+	K8sUseJSONPatch = "k8s-use-json-patch"
```

Comment:
Rename this to `k8s-force-json-path` so it's clear that's in not an opt-in but a force flag.